org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/component/ProcessInfoDialog.java

```diff
@@ -216,6 +242,64 @@ public void onEvent(Event event) throws Exception {
 		if (event.getTarget() == btnOk) {
 			this.detach();
 		}
+		if(event.getTarget() == btnPrint) {
+			Clients.showBusy(Msg.getMsg(Env.getCtx(), "Processing"));
+			Clients.response(new AuEcho(ProcessInfoDialog.this, "onPrint", null));

```

Comment:
the alternative is to use `Executions.schedule(this.getDesktop(), e -> onPrint(), new Event("onPrint"));`. The benefit is you can declare onPrint as private and didn't add a "fake" public method to ProcessInfoDialog (fake as in although it is public but it is not means to be call directly by other class).